src/app/services/main-form-manager/models-config.ts

```diff
@@ -25,7 +25,11 @@ import { FieldsConfiguration } from 'arlas-web-contributors';
 export const JSONPATH_COUNT = '$.count';
 export const JSONPATH_METRIC = '$.metrics[0].value';
 export const CHIPSEARCH_TYPE = 'chipssearch';
+export const SEARCH_TYPE = 'search';
+// Deprecated
```

Comment:
If you put it like `/** @deprectaed */` it will show up when hovering the variable